### DISCUSSION TITLE
Hex vs. other boards

### DISCUSSION THREAD

**Discussion by graue** [gereleth/hexapipes#99] [General]
Hi, thanks for making this puzzle! Just some feedback that while the other board types can be an interesting diversion now and then, for me the original hex type is the most fun. It feels "just right" between the complexity of the octagons and the simplicity of the squares/cubes/elongated triangles. My personal preference would be for more dailies to be hexes.

**Comment by gereleth** [comment]
Hi, thanks for the feedback. Though I'm astonished you call etrats simple, they're the hardest puzzle kind by far.

I usually alternate between playing octagons and etrats these days. But my own personal preference regarding dailies would be to just stop them altogether. Curse my past self who saddled me with this chore. =)

I'm working on a more automated way of creating dailies to free me from this task. I can bias it more towards hexagons no problem.

Alternatively if anyone volunteers to set daily puzzles I'll gladly show them the ropes and accept PRs with new dailies)).
